NEWS.md

```
@@ -97,7 +104,7 @@ pre-game line is missing.
 
 # **cfbfastR v3.0.0**
 
-### New release-dataset loaders (39 functions)
+### New release-dataset loaders (48 functions)
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)